packages/app/src/components/drawer/accounts/create-account.tsx

```diff
@@ -42,7 +42,14 @@ export const CreateAccount: React.FC<CreateAccountProps> = ({ close }) => {
         <Text fontSize={2}>Your new account has been created.</Text>
       </Box>
       <Flex width="100%" flexGrow={1} mt="base">
-        <Button width="100%" ml={2} onClick={close}>
+        <Button
+          width="100%"
+          ml={2}
```

Comment:
why margin left? this seems like a mistake, the parent should be handling this space. Additionally this should be `tight` not `2`

Reply:
Hm, the `ml` was there before so I didn't think about it. It's definitely just a bug. I've removed it.